src/ClassicUO.IO/DataReader.cs

```diff
@@ -56,8 +56,23 @@ public unsafe class DataReader
 
         public bool IsEOF => Position >= Length;
 
+        ~DataReader() {
```

Comment:
FYI - We use the standard microsoft formatting (I know it is unusual), with the `{` on the next line.

I can fix this, squash it, and merge it.

Reply:
I missed that one, I can fix it easily. I have stgit superpowers!